vault/dendron.topic.lookup.find.md

```diff
@@ -17,6 +17,15 @@ Below is an example of a simple two level hierarchy with notes on various [comma
 
 ![](https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/images/lookup-cli.gif)
 
+## Sort Ordering
+When there are multiple matches they will be sorted in the following grouped order:
+  1. By FuseJS match score.
+  1. By whether the note stub or not (stubs go to the bottom of the score group).
```

Comment:
nit: 'By whether the note _is a_ stub or not'

Reply:
If we know a good spell checker extension, we should add a suggestion in our https://wiki.dendron.so/notes/301e4129-6933-4be7-a4bd-8125171360d8/ extension page (If that is the page where we expect to find extra extensions to add into VS Code)